packages/ui-gui/package.json

```diff
@@ -1,28 +1,38 @@
 {
   "name": "@webpack-cli/ui-gui",
   "version": "0.0.1",
+  "license": "MIT",
   "description": "GUI for webpack-cli ui",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "webpack --config=webpack.prod.js",
-    "serve": "webpack-dev-server --config=webpack.prod.js"
+    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
+    "build": "preact build",
+    "serve": "preact build && preact serve",
+    "dev": "preact watch",
+    "lint": "eslint src"
   },
-  "license": "MIT",
+  "eslintConfig": {
+    "extends": "eslint-config-synacor",
```

Comment:
Why this config? I am curios, I've never seen it

Reply:
@ematipico this was generated by `preact-cli` booilerplate.